dotnet-statsig/src/Statsig/Server/Evaluation/Evaluator.cs

```diff
@@ -64,6 +64,178 @@ internal ConfigEvaluation GetLayer(StatsigUser user, string layerName)
+            {
+                if (kv.Value.Entity == "segment") 
+                {
+                    continue;
+                }
+
+                var hashedName = HashName(kv.Value.Name);
+                var gate = Evaluate(user, kv.Value).GateValue;
+                var entry = new Dictionary<string, object>
+                {
+                    ["name"] = hashedName,
+                    ["value"] = gate.Value,
+                    ["rule_id"] = gate.RuleID,
+                };
+                if (gate.SecondaryExposures.Count > 0)
+                {
+                    entry["secondary_exposures"] = CleanExposures(gate.SecondaryExposures);
+                }
+                gates.Add(hashedName, entry);
+            }
+            
+            var dynamicConfigs = new Dictionary<string, Dictionary<string, object>>();
+            foreach (var kv in _store.DynamicConfigs)
+            {
+                var hashedName = HashName(kv.Value.Name);
+                var config = Evaluate(user, kv.Value).ConfigValue;
+                var entry = ConfigSpecToInitResponse(hashedName, kv.Value, config);
+                if (kv.Value.Entity != "dynamic_config")
+                {
+                    entry["is_experiment_active"] = IsExperimentActive(kv.Value);
+                    entry["is_user_in_experiment"] = IsUserAllocatedToExperiment(user, kv.Value);
+                }
+                dynamicConfigs.Add(hashedName, entry);
+            }
+
+            var layerConfigs = new Dictionary<string, Dictionary<string, object>>();
+            foreach (var kv in _store.LayerConfigs)
+            {
+                var hashedName = HashName(kv.Value.Name);
+                var evaluation = Evaluate(user, kv.Value);
+                var config = evaluation.ConfigValue;
+                var entry = ConfigSpecToInitResponse(hashedName, kv.Value, config);
+
+                if (!string.IsNullOrWhiteSpace(evaluation.ConfigDelegate))
+                {
+                    entry["allocated_experiment_name"] = HashName(evaluation.ConfigDelegate);
+                    entry["is_experiment_active"] = true;
+                    entry["is_user_in_experiment"] = true;
+                }
+                if (!entry.ContainsKey("explicit_parameters")) 
+                {
+                    entry["explicit_parameters"] = new List<string>();
+                }
+                entry["undelegated_secondary_exposures"] = 
+                    CleanExposures(evaluation.UndelegatedSecondaryExposures);
+                layerConfigs.Add(hashedName, entry);
+            }
+
+            var result = new Dictionary<string, Object>
+            {
+                ["feature_gates"] = gates,
+                ["dynamic_configs"] = dynamicConfigs,
+                ["layer_configs"] = layerConfigs,
+                ["sdkParams"] = new Object(),
+                ["has_updates"] = true,
+                ["time"] = 0,
```

Comment:
If this is replacing initialize, probably best to set the time?  For the node SSR, we wanted the values to be overriden potentially by another request from the client, but for this case, I we should probably set the time.

Reply:
Done.